### ISSUE TITLE
Sway does not recognize i915 iGPU

### ISSUE THREAD

**Issue by maholloway** [swaywm/sway#3948]
* Sway Version:
`sway -v` Output:
sway version  (2018-07-26, branch "")
Obtain a debug log like so:

* Debug Log:
sway.log and `dmesg | tail` output
https://gist.github.com/maholloway/ee8dc8f27210cd190bf4d1f28c9377d0

* Configuration File:
Reproduced successfully with stock config (Moving .config/sway/config out of expected path does not resolve the issue)

Steps to reproduce:
Install sway via Fedora 28 dnf
Choose sway from GDM login session selector OR launch from tty
Run swaymsg -t get_outputs and observe one output presented instead of two

I have attempted to follow #2889 troubleshooting steps such as setting WLR_DRM_DEVICES to include both the dGPU and iGPU, the issue still persists.

Any more info needed just ask :)

**Comment by ascent12** [comment]
You're using sway 0.15, which is not supported any more.

**Comment by maholloway** [comment]
Problem solved with an upgrade to 1.0 as built in https://copr.fedorainfracloud.org/coprs/tofik/sway/

Steps to solve (F28):
```
sudo dnf copr enable tofik/sway
sudo dnf --best --allowerasing install libwayland-\*
sudo dnf --best install wlroots
sudo dnf install sway
```

I also rebooted before I verified the fix but it's likely not necessary.
